robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -580,6 +580,22 @@ def clear_lookup_value(self,field):
         self.selenium.scroll_element_into_view(locator)
         self.selenium.click_button(locator)
 
+    def select_value_from_lookup(self, value):
+        """Selects the first value from a lookup field"""
+        locator=npsp_lex_locators["gift_entry"]["payment_imported_lookup"]
+        self.selenium.wait_until_page_contains_element(locator)
+        self.selenium.execute_javascript("window.scrollBy(0,200)")
+        self.selenium.scroll_element_into_view(locator)
+        try:   
+            self.selenium.click_element(locator)
+        except ElementClickInterceptedException:
```

Comment:
your condition for try/except is the same . Shouldn't  it differ ?

Reply:
fixed it